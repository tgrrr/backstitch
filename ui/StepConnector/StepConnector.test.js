import React from 'react'
import renderer from 'react-test-renderer'
import { StepConnector } from '.'

it('StepConnector: default', () => {
    const component = renderer.create(
        <StepConnector>
            <>
                StepConnector
            </>
        </StepConnector>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
