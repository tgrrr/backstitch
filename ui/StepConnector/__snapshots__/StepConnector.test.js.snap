// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StepConnector: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Stepconnector "
    data-testid="Stepconnector"
  >
    <div
      className="MuiStepConnector-root MuiStepConnector-horizontal"
    >
      <span
        className="MuiStepConnector-line MuiStepConnector-lineHorizontal"
      />
    </div>
  </div>
</div>
`;
